indico/modules/events/registration/forms.py

```diff
@@ -536,7 +536,7 @@ def __init__(self, event, **kwargs):
         all_templates = set(event.designer_templates) | get_inherited_templates(event)
         badge_templates = [tpl for tpl in all_templates if tpl.type.name == 'badge']
         signals.event.filter_selectable_badges.send(type(self), badge_templates=badge_templates)
-        tickets = kwargs.pop('tickets')
+        tickets = kwargs.get('tickets')
```

Comment:
That change does not look correct. Random non-field-related data should not be passed to the form constructor.

Reply:
I realised I could get the tickets from the kwargs in the patched class, so dropping this change.